pkgs/development/libraries/libxml2/default.nix

```diff
@@ -47,11 +50,22 @@ stdenv.mkDerivation rec {
     ./utf8-xmlErrorFuncHandler.patch
   ];
 
-  strictDeps = true;
-
   nativeBuildInputs = [
     pkg-config
-    autoreconfHook
+    (autoreconfHook.override {
+      automake = automake.overrideAttrs (attrs: {
+        patches = attrs.patches or [ ] ++ [
+          # Unable to find Python 3.10
+          (fetchpatch {
+            url = "https://github.com/autotools-mirror/automake/commit/930a9a73f4bb776ec334eff4cf6e182802841daa.patch";
```

Comment:
Please love this to automake. This is a general, annoying issue.

Reply:
I am unable to reproduce it any more.